src/main/java/net/snowflake/ingest/utils/ParameterProvider.java

```diff
@@ -38,6 +38,8 @@ public class ParameterProvider {
   public static final String MAX_CHUNKS_IN_BLOB = "MAX_CHUNKS_IN_BLOB".toLowerCase();
   public static final String MAX_CHUNKS_IN_REGISTRATION_REQUEST =
       "MAX_CHUNKS_IN_REGISTRATION_REQUEST".toLowerCase();
+  public static final String BLOB_UPLOAD_TIMEOUT_IN_SEC =
+      "BLOB_UPLOAD_TIMEOUT_IN_SEC".toLowerCase();
```

Comment:
since this is user facing lets make it "in_seconds" instead of "in_sec"

Reply:
Fixed.